src/plugins/cams/datasource.py

```
@@ -0,0 +1,509 @@
+
+class CAMSDataSource(DataSource):
+    """Data source for CAMS atmospheric composition data.
+    
+    Provides gridded PM2.5, PM10, and other pollutant data from:
+    - EAC4 Reanalysis: Historical data with 3-hour resolution
+    - Operational Forecasts: 5-day forecasts updated twice daily
+    
+    Coverage matches OpenMeteo grid: 24-46°N, 123-146°E
+    """
+    
+    def __init__(
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)